### ISSUE TITLE
Error during parsing of paket.lock

### ISSUE THREAD

**Issue by JohanLarsson** [fsprojects/Paket#2437]
### Description

Getting ~build errors after running `paket install -f`

```
Severity	Code	Description	Project	File	Line	Suppression State
Error		Error during parsing of C:\Git\Gu.Roslyn.Asserts\paket.lock.
Message: unknown lock file format RESTRICTION: || (== net461) (== netstandard13)		C:\Git\Gu.Roslyn.Asserts\Gu.Roslyn.Asserts.Tests\paket.references	0	
```

### Repro steps

paket.dependencies

```
source https://www.nuget.org/api/v2/
framework: netstandard1.3, net461
redirects: on

nuget Microsoft.CodeAnalysis.CSharp.Workspaces 2.2.0
nuget NUnit

group Analyzers
  source http://www.nuget.org/api/v2/
  nuget Gu.Analyzers
  nuget StyleCop.Analyzers
  nuget AsyncUsageAnalyzers
```

**Comment by matthid** [comment]
Are you using Visual Studio paket extension? Somewhere an old paket version is trying to read paket 5 lockfile...

**Comment by forki** [comment]
We should update Paket.VisualStudio....

**Comment by JohanLarsson** [comment]
@matthid That is probably it! Did not think about it but think I have it installed.

**Comment by JohanLarsson** [comment]
Can the extension use magic mode?

**Comment by forki** [comment]
Unfortunately the extension uses a dll. That's a real design issue and I
still hope someone will finally fix it and let it use command line interop

Am 17.06.2017 9:40 nachm. schrieb "Johan Larsson" <notifications@github.com

**Comment by forki** [comment]
ok just released a new plugin version. please test

**Comment by JohanLarsson** [comment]
Works!

**Comment by Rajivhost** [comment]
Hi, I use the latest version of the plugin and still have the issue.

**Comment by forki** [comment]
I just released a new addin 0.33 which targets paket 5.85

Am 10.08.2017 1:51 nachm. schrieb "Rajiv Mounguengue" <
notifications@github.com>:
